linux_os/guide/services/obsolete/inetd_and_xinetd/configure_etc_hosts_deny/rule.yml

```diff
@@ -10,6 +10,10 @@ description: |-
     The following line in the file ensures that access to services supporting this mechanism is denied to any clients
     not mentioned in <tt>/etc/hosts.allow</tt>:
     <pre>ALL: ALL</pre>
+    It is advised to inspect available network services which might be affected by modification of file mentioned above prior to performing the remediation of this rule.
+    If there exist services which might be affected and access to them should not be blocked,
+    modify the <tt>/etc/hosts.allow</tt> file appropriately before performing the remediation.
+
```

Comment:
Isn't this part redundant with the warning?

Reply:
It is similar, but does it hurt? Are the warnings and descriptions visible in similar way in terms of visual style? I just don't want users to miss the information that if they want to allow some services, they have to place them into hosts.allow.

Reply:
>  if they want to allow some services, they have to place them into hosts.allow.

I believe this is already covered in the description, starting from line 10.

Reply:
I see you removed it from the warning, I'm fine with that.